src/main/java/chess/controller/web/WebGameController.java

```diff
@@ -0,0 +1,106 @@
+package chess.controller.web;
+
+import static spark.Spark.get;
+import static spark.Spark.post;
+
+import chess.domain.piece.PieceColor;
+import chess.domain.dto.MovableDto;
+import chess.domain.dto.ResponseDto;
+import chess.domain.dto.ResponseStatusDto;
+import chess.domain.game.ChessGame;
+import chess.domain.game.Status;
+import chess.service.ChessService;
+import com.google.gson.Gson;
+import java.util.HashMap;
+import java.util.Map;
+import spark.ModelAndView;
+import spark.template.handlebars.HandlebarsTemplateEngine;
+
+public class WebGameController {
+
+    private final ChessService chessService;
+
+    public WebGameController() {
+        this.chessService = new ChessService();
+    }
+
+    private static String render(Map<String, Object> model, String templatePath) {
+        return new HandlebarsTemplateEngine().render(new ModelAndView(model, templatePath));
+    }
+
+    public void run() {
+        Gson gson = new Gson();
+
+        get("/", (req, res) -> {
+            return render(new HashMap<>(), "index.html");
+        });
+        get("/load", (req, res) -> {
+            return gson.toJson(load());
+        });
+        get("/start", (req, res) -> {
+            return gson.toJson(start());
+        });
+        post("/move", (req, res) -> {
+            final MovableDto movableDto = gson.fromJson(req.body(), MovableDto.class);
+            return gson.toJson(move(movableDto));
+        });
+        get("/status", (req, res) -> {
+            return gson.toJson(status());
+        });
+        get("/end", (req, res) -> {
+           return gson.toJson(end());
+        });
+    }
+
+    private ResponseDto load() {
+        ChessGame chessGame = null;
+        try {
+            chessGame = chessService.load();
+            return ResponseDto.createResponseDto(chessGame.turn().name(), chessGame);
+        } catch (IllegalArgumentException e) {
+            return ResponseDto.createErrorResponseDto(e.getMessage(), chessGame);
+        }
+    }
+
+    private ResponseDto start() {
+        ChessGame chessGame = null;
+        try {
+            chessGame = chessService.createRoom();
+            return ResponseDto.createResponseDto(chessGame.turn().name(), chessGame);
+        } catch (IllegalArgumentException e) {
```

Comment:
chessGame만 넘겨줘도 될 것 같네요 :)
```suggestion
        try {
            chessGame = chessService.createRoom();
            return ResponseDto.createResponseDto(chessGame);
        } catch (IllegalArgumentException e) {
```